browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1216,14 +1218,42 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			contentDivs.forEach(function(tabPage)
 			{
 				tabPage.addEventListener('keydown', function(e) {
-					if (['ArrowUp', 'ArrowDown', 'ArrowLeft', 'ArrowRight'].includes(e.key)) {
+					// Determine key direction
```

Comment:
hmm, how this code landed here :)     https://github.com/CollaboraOnline/online/pull/7194
we should inherit from it and create separate notebookbar tab handler, not add more ifs for custom element in generic handler: todo later

wrt `if (data.tabs && data.parent.id === 'NotebookBar') {`